packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -343,55 +315,6 @@ test.serial('bulk operation BULK_GRANULE applies workflow to list of granules',
   });
 });
 
-test.serial('bulk operation BULK_GRANULE applies workflow to granules returned by query', async (t) => {
```

Comment:
This test and all the other tests are still needed for unit testing es query against Cloud Metrics.